fix: Fix options for Node UnhandledRejection integration (#11245)
getsentry/sentry-docs: docs/platforms/javascript/common/configuration/integrations/unhandledrejection.mdx

## Options

### `mode`

_Type: `'none' | 'warn' | 'strict'`_

> This option defines what to do after capturing an unhandled rejection.
> It that mimicks behavior of [node's](https://nodejs.org/api/cli.html#--unhandled-rejectionsmode) `--unhandled-rejection` flag:

Reviewer: ```suggestion
This option defines what to do after capturing an unhandled rejection and
mimics the behavior of [node's](https://nodejs.org/api/cli.html#--unhandled-rejectionsmode) `--unhandled-rejection` flag:
```
